examples/oktapam_gw_setup/gw_setup_example.tf

```diff
@@ -0,0 +1,69 @@
+terraform {
+  required_version = ">= 0.12.0"
+
+  required_providers {
+    oktapam = {
+      version = "0.1.4"
+      source  = "okta.com/pam/oktapam"
+    }
+    google = {
+      source = "hashicorp/google"
+      version = "4.30.0"
+    }
+  }
+}
+
+provider "google" {
+    # Configuration options
+    project	= "asa-demo-316514"
+    region	= "us-central1"
+    zone	= "us-central1-a"
+}
+
+# Create gateway setup token
+resource "oktapam_gateway_setup_token" "test-gw" {
+  description          = "tf-gw"
+  labels               = {env:"prod"}
+}
+
+output "gw_id" {
```

Comment:
We can move this to separate file outputs.tf.

Reply:
Done thanks